internal/tools/aws/constants.go

```diff
@@ -132,6 +132,10 @@ const (
 	// zone ID in AWS Route53.
 	DefaultPublicCloudDNSTagValue = "public"
 
+	// HibernatingInstallationResourceRecordIDPrefix is a prefix given to AWS
+	// route53 resource records when the installation it points to is hibernating.
+	HibernatingInstallationResourceRecordIDPrefix = "[hybernating] "
```

Comment:
typo: "hybernating" instead of "hibernating"

Reply:
lol, time for `[hypernating]` installations I guess.